package.json

```diff
@@ -28,6 +28,7 @@
     "ts-loader": "^8.0.5"
   },
   "scripts": {
+    "preinstall": "if [[ $npm_execpath =~ 'yarn' ]]; then echo 'Use NPM!' && exit 1; fi",
```

Comment:
😆

Reply:
I feel like this is the only repo using NPM, and I always trip up!

Reply:
Discussing this with @sndrs, there might be issues when installing this dependencies in `yarn` projects.